### ISSUE TITLE
Why times are all different? Only "google.com" returns accurate time!

### ISSUE THREAD

**Issue by ghost** [ioerror/tlsdate#195]
tlsdate -n -s -H SEREVER.FQDN.HERE -p 443 -v -V

Time is correct:
www.google.com

Time is TERRIBLE:
mail.riseup.net (UTC 1941!?)
www.torproject.org (UTC 2029!)
www.nsa.gov

When I visit these websites using a browser, it returns correct "Date:".
Why tlsdate can't detect it?

**Comment by oniGino** [comment]
I'm my understanding that as an additional security measure TLS(all versions?) now randomize the date+time it sends.. so effectively killing this project... which is really too bad since ntp really needs an solid alternative

Here is a quick blurb that sums it up well
 https://security.stackexchange.com/questions/71364/tls-reliance-on-system-time

**Comment by AlexeySalmin** [comment]
>  which is really too bad since ntp really needs an solid alternative

That is true :( If anyone discovers an alternative please share a link here so that subscribed users can see it.
